src/types/Category.ts

```diff
@@ -1,21 +1,25 @@
-import type { CultureRef } from './Culture';
+import type { Culture, CultureRef } from './Culture';
 
 export interface CategoryRef {
     id: number;
     display_name: string;
     display_description: string | null;
-    i18n: {
-        [localeCode: CultureRef['code']]: {
-            description: string | null;
-            locale: CultureRef;
-            name: string;
-            slug: string | null;
-        };
-    };
+    i18n: Partial<{
```

Comment:
I've also made it use `Partial<...>` to hint typescript that not every locale code is defined.

Without it, Typescript is always happy seeing `category['nl'].name`, even though the `nl` locale may not be present in the `i18n` object.